src/_cffi_src/openssl/ssl.py

```diff
@@ -652,4 +657,33 @@
 static const long TLS_ST_BEFORE = 0;
 static const long TLS_ST_OK = 0;
 #endif
+
+#ifndef OPENSSL_NO_DTLS
+static const long Cryptography_HAS_DTLS = 1;
+/* Wrap DTLSv1_get_timeout to avoid cffi to handle a 'struct timeval'. */
+long DTLSv1_get_timeout_wrapped(
+                                SSL *ssl,
+                                time_t *ptv_sec,
+                                long int *ptv_usec)
+{
+    struct timeval tv = { 0 };
```

Comment:
Is this declaration safe in C89 for Windows? I think so, but want someone to double check me.